.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue is requesting
+a new topic, please preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics, and the creation of new topics.
+
+This way, we have those discussions before significant
```

Comment:
Shouldn't new topics be a different section? What's the purpose of this doc? For all issues? Don't you want a new topic section? It would be good to have links to each section at the top. This text should be written as:

something like:

An outline and discussion should take place before significant effort on a topic.